src/main/java/org/jenkinsci/plugins/GithubAuthenticationToken.java

```diff
@@ -377,17 +387,26 @@ public GHOrganization loadOrganization(String organization) {
         return null;
     }
 
-    public GHRepository loadRepository(String repositoryName) {
-        try {
-            if (gh != null && isAuthenticated()) {
-                return getGitHub().getRepository(repositoryName);
-            }
-        } catch (IOException e) {
-            LOGGER.log(Level.WARNING,
-                    "Looks like a bad GitHub URL OR the Jenkins user does not have access to the repository{0}",
-                    repositoryName);
-        }
-        return null;
+    public GHRepository loadRepository(final String repositoryName) {
```

Comment:
Why is this a `final String` as opposed to just `String`?

Reply:
To be able to access it inside the anonymous Callable below.

Reply:
I'm not sure I understand the nuance why `final` is required to access it from the anonymous callable.  Does that mean `isPublicRepository(final String repositoryName)` should be updated to `isPublicRepository(String repositoryName)`?

Reply:
@samrocketman you're totally right. As the anonymous callable was moved, the `isPublicRepository` parameter does not need to be `final` anymore.

**Note:** when this plugin will support Java 8, the `final` could be removed since the parameter will be considered [effectively final](https://stackoverflow.com/questions/20938095/difference-between-final-and-effectively-final).